dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -1702,6 +1719,9 @@ def test_text_profiler_results(self):
                 }
             }
         }
+        # pop out the capacity to test first
+        self.assertAlmostEqual(expected_report['global_stats'].pop('capacity'),
+                               self.report['global_stats'].pop('capacity'))
```

Comment:
not blocking, btu does this still need to be popped or `assertAlmostEqual`

Reply:
removed